tests/managers/index.js

```diff
@@ -32,10 +40,19 @@ describe('ManagerTests', function () {
                 voiceCommandUpdateOperationTests(appClient);
```

Comment:
Something with the most recent update is causing failures in the `voiceCommandManagerTests` and `voiceCommandUpdateOperationTests` for me. I've tested this with manticore across multiple browsers. Everything passes if these tests are moved above the `textAndGraphicManagerTests` and run first though.

Reply:
That's annoying. I have moved those tests and got the same results now.